docs/chainbridge/setup/setup-erc20-transfer.md

```diff
@@ -0,0 +1,148 @@
+  # Set Resource ID for ERC20
+  --resourceId "0x000000000000000000000000000000c76ebe4a02bbc34786d860b355f5a5ce00" \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --handler "[ERC20_HANDLER_CONTRACT_ADDRESS]" \
+  --targetContract "[ERC20_CONTRACT_ADDRESS]"
+
+# For Polygon SDK chain
+$ cb-sol-cli bridge register-resource \
+  --url http://localhost:10002 \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  # Set Resource ID for ERC20
+  --resourceId "0x000000000000000000000000000000c76ebe4a02bbc34786d860b355f5a5ce00" \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --handler "[ERC20_HANDLER_CONTRACT_ADDRESS]" \
+  --targetContract "[ERC20_CONTRACT_ADDRESS]"
+```
+
+## (Optional) Make contracts mintable/burnable
+
+When you send ERC20 tokens between chains, tokens are processed by either way as follows:
+
+(1) Lock/Release mode
+
+**Source chain**: The tokens you're sending will be locked in ERC20 Handler Contract  
+**Destination chain**: The same amount of tokens as you sent in the source chain would be unlocked and transferred from ERC20 Handler contract to the recipient account in the destination chain.
+
+(2) Burn/Mint mode
+
+**Source chain**: The tokens you're sending will be burned  
+**Destination chain**: The same amount of tokens as you sent in the source chain would be minted in the destination chain and sent to the recipient account.
+
+You can use different modes in each chain. It means that you can lock ERC20 token in the main chain while mint ERC20 token in the sub chain for transfer. For instance, it may make sense to lock/release tokens if the total supply or mint schedule is controlled. Tokens would be minted/burned if the contract in the sub chain has to follow the supply in the main chain.
+
+The default mode is **lock/release** mode. If you want to make the Tokens mintable/burnable, you need to call `adminSetBurnable` method. If you want to mint tokens on execution, you will need to grant `minter` role to ERC20 Handler contract.
+
+```bash
+# Let ERC20 contract burn on source chain and mint on destination chain
+$ cb-sol-cli bridge set-burn \
+  --url http://localhost:10002 \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --bridge "[BRIDGE_CONTRACT_ADDRESS]" \
+  --handler "[ERC20_HANDLER_CONTRACT_ADDRESS]" \
+  --tokenContract "[ERC20_CONTRACT_ADDRESS]"
+
+# Grant minter role to ERC20 Handler contract
+$ cb-sol-cli erc20 add-minter \
+  --url http://localhost:10002 \
+  --privateKey [ADMIN_ACCOUNT_PRIVATE_KEY] \
+  --erc20Address "[ERC20_CONTRACT_ADDRESS]" \
+  --minter "[ERC20_HANDLER_CONTRACT_ADDRESS]"
+```
+
+## Step 2: Transfer ERC20 Token
+
+We will send ERC20 Tokens from Polygon PoS chain to Polygon SDK chain.
+
+First, you will get tokens by minting. An account with `minter` role can mint new tokens. The account that has deployed ERC20 contract has `minter` role as default. For other accounts, you would need to run `cb-sol-cli erc20 add-minter` command to grant a minter role to the account before minting.
```

Comment:
```suggestion
First, you will get tokens by minting. An account with the `minter` role can mint new tokens. The account that has deployed the ERC20 contract has the `minter` role by default. To specify other accounts as members of the `minter` role, you need to run the `cb-sol-cli erc20 add-minter` command.
```

Reply:
Specify also how often this needs to happen so that the users may know if this is something that can be done in advance to minimize gas costs

Reply:
Fixed at 5756c760532fdfa7da42c47aad4dc893d1e4074e

Reply:
@lazartravica Sorry. To be honest, I don't understand this well. In actual case, the user who wants to transfer tokens between chains won't mint tokens because the user should not be allowed to mint. Basically, the manager of the token contract will mint or the token contract schedules to mint.